web/source/kmwbase.ts

```diff
@@ -159,21 +159,22 @@ class KeymanBase {
   _GetEventObject<E extends Event>(e: E) {  // I2404 - Attach to controls in IFRAMEs
     if (!e) {
       e = window.event as E;
-      if(!e) {
-        var elem: HTMLElement|Document = this.domManager.getLastActiveElement();
-        if(elem) {
-          elem = elem.ownerDocument;
-          var win: Window;
-          if(elem) {
-            win = elem.parentWindow;
-          }
-          if(!win) {
-            return null;
-          }
+      // if(!e) {
```

Comment:
I see this is all commented out. Can you explain what is happening here? Is this function obsolete?